code/modules/vehicles/vehicle_key.dm

```diff
@@ -24,7 +24,7 @@
 	icon_state = "keyjanitor"
 
 /obj/item/key/janitor/suicide_act(mob/living/carbon/user)
-	switch(user.mind.get_skill_level(/datum/skill/cleaning))
+	switch(user?.mind?.get_skill_level(/datum/skill/cleaning))
```

Comment:
```suggestion
	switch(user.mind?.get_skill_level(/datum/skill/cleaning))
```
Unless you know of a case where `user` might not exist here, don't add silent sanity checks that might hide errors.